src/deprecated/event.js

```diff
@@ -30,13 +30,12 @@ jQuery.fn.extend( {
 
 jQuery.each( ( "blur focus focusin focusout resize scroll click dblclick " +
 	"mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave " +
-	"change select submit keydown keypress keyup contextmenu" ).split( " " ),
-	function( _i, name ) {
-
-		// Handle event binding
-		jQuery.fn[ name ] = function( data, fn ) {
-			return arguments.length > 0 ?
-				this.on( name, null, data, fn ) :
-				this.trigger( name );
-		};
-	} );
+	"change select submit keydown keypress keyup contextmenu" ).split( " " ), function( _i, name ) {
```

Comment:
This line is a bit long; could we leave it as it was before?

Reply:
I will modify it again to minimize code changes.